### PR TITLE
[OSCD] Updated win_etw_trace_evasion - Added new detections, Removed reference to deprecated rule and changed selections

### PR DESCRIPTION
**Pull request by w0rk3r**
Just working on an atomic test for this rule and saw room for improvement

### CODE REVIEW

File: rules/windows/process_creation/win_etw_trace_evasion.yml [hunk 1]
```diff
@@ -19,13 +19,38 @@ logsource:
...
+        Commandline|contains|all:
+          - 'Set-EtwTraceProvider'
+          - '{1418ef04-b0b4-4623-bf7e-d74ab47bbdaa}'
+          - 'EventLog-Microsoft-Windows-WMI-Activity-Trace'
+          - '0x11'
+    selection_disable_5: #ETW provider removal from a trace session
+        Commandline|contains|all:
+          - "logman" 
+          - "update trace" 
+          - "Trace --p"
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_etw_trace_evasion.yml:52]: Hello @w0rk3r! Great job with the rule, it mostly looks great. There is only one tiny issue — the spaces here:

```
          - "update trace" 
          - "Trace --p"
```

Could you please remove them, splitting the strings into separate parts, like that:

```
          - "update"
          -  "trace" 
          - "Trace"
          - "--p"
```

I would ask the maintainers to merge the PR after that.
Thank you!

Comment (w0rk3r) [reply rules/windows/process_creation/win_etw_trace_evasion.yml]: Updated the rule, just kept one trace as it would match the two variants anyway, ty!

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_etw_trace_evasion.yml]: awesome! thank you!

**Comment by yugoslavskiy** [comment]
Thank you very much for your continuous contribution, @w0rk3r! You did a great job!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**